src/components/image-container/image-container.scss

```diff
@@ -39,3 +50,15 @@
 		display: block;
 	}
 }
+
+.image-container__image--object {
+	height: auto;
+	max-height: 100%;
+	max-width: 100%;
+	// TODO: we can't use these declarations until we have a BE that can tell us the basic dimensions for markup :( 
+	// object-fit: scale-down;
+	// object-position: center;
+	object-fit: unset;
+	object-position: unset;
```

Comment:
Great question. I believe we do because of: 
https://github.com/metmuseum/marble/blob/27152b6df2a13e312a468abdbc2ba508de182f51/src/components/image-container/image-container.scss#L22-L23

(I assumed the usage would be `class="image-container__image image-container_image--object"`)

But really wanted you and @JosephCurley to give input on that? 🤔

Reply:
Yeah I'm kind of wondering if we can leave those 2 rules out as well? I tried removing them in the console and didn't see any problems in any of the images, but I could be missing something in the whole system.

Reply:
I tried this on https://github.com/metmuseum/marble/commit/a17130f4e56bdf170f66327e3ee70360dc12e941 and you are correct - there is no visible change in the snapshots on Chrome or locally on Firefox and Safari. @JosephCurley what do you think? Good to remove these `unset` declarations here and let them inherit declarations that also don't change them? I think you actually asked that the original class have `cover` etc?

Reply:
Then I think we could also remove one of those classes, no? `class="image-container__image image-container_image--object"` It's not a big deal but it would make it simpler.

Reply:
Oh really? I think what happened was, I originally wrote this new container with `srcset` and `sizes` attrs on the `img`s, but realized responsive images would have to be addressed later, because again the whole point was playing it safe about pixel dimensions.  so maybe the switch back to only having a `src` made this entire class unnecessary?! will check now...

Reply:
ok i checked we def need some of those properties lol `https://www.chromatic.com/test?appId=5ef272f9ab690c0022ef30ab&id=628678beb179cd004aab4564`

Reply:
yeah those properties are fine to leave in - they only do anything in a pretty unique situation